### PR TITLE
fix: [CDS-34691]: message for deleted template case added

### PR DESCRIPTION
**Pull request by samarth-sharma31**
##### Summary:

When the template has already been deleted via a duplicate tab, right drawer open up with no details. A message has been added for this scenario 

##### Jira Links:

https://harness.atlassian.net/browse/CDS-34691

##### Screenshots:

https://user-images.githubusercontent.com/96409598/157625613-fb40fa04-6d49-4785-af24-5212e7bfda3e.mov

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by samarth-sharma31** [comment]
retrigger titlecheck

**Comment by samarth-sharma31** [comment]
retrigger titlecheck

**Comment by samarth-sharma31** [comment]
retrigger jest

**Review by yc147 — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

File: src/modules/72-templates-library/strings/strings.en.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ alwaysUseStableVersion: Always use the stable version
 inputsCount: 'Total Inputs: {{count}}'
 templateUpdatedError: The Template was updated by another user. If you update the Template their changes will overwrite your changes. Would you like to update the Template?
 openInTemplateStudio: Open in Template Studio
+templateHasBeenDeleted: The selected template has been deleted. Please refresh page.
```
Comment (chiragp1212) [inline src/modules/72-templates-library/strings/strings.en.yaml:19]: please refresh the page

**Comment by samarth-sharma31** [comment]
trigger cypress

**Comment by samarth-sharma31** [comment]
trigger cypress

**Review by yc147 — APPROVED** [review]

**Review by chiragp1212 — APPROVED** [review]

**Pull request merged**